sdk/python/feast/feature_store.py

```diff
@@ -870,46 +878,16 @@ def get_online_features(
                     feature_name
                 ] = GetOnlineFeaturesResponse.FieldStatus.PRESENT
 
-        # Note: each "table" is a feature view
         for table, requested_features in grouped_refs:
-            entity_keys = _get_table_entity_keys(
-                table, union_of_entity_keys, entity_name_to_join_key_map
-            )
-            read_rows = provider.online_read(
-                config=self.config,
-                table=table,
-                entity_keys=entity_keys,
-                requested_features=requested_features,
+            self._populate_result_rows_from_feature_view(
+                entity_name_to_join_key_map,
+                full_feature_names,
+                provider,
+                requested_features,
+                result_rows,
+                table,
+                union_of_entity_keys,
             )
```

Comment:
Was this just a refactoring?

Reply:
Had to because our linter complains about complex methods